lxd/apparmor/qemuimg.go

```diff
@@ -123,56 +120,41 @@ func QemuImg(sysOS *sys.OS, cmd []string, imgPath string, dstPath string) (strin
 }
 
 // qemuImgProfileLoad ensures that the qemu-img's policy is loaded into the kernel.
-func qemuImgProfileLoad(sysOS *sys.OS, imgPath string, dstPath string, allowedCmdPaths []string) error {
-	profileName := getProfileName(imgPath, dstPath)
+func qemuImgProfileLoad(sysOS *sys.OS, imgPath string, dstPath string, allowedCmdPaths []string) (string, error) {
+	name := fmt.Sprintf("<%s>_<%s>", strings.ReplaceAll(strings.Trim(imgPath, "/"), "/", "-"), strings.ReplaceAll(strings.Trim(dstPath, "/"), "/", "-"))
```

Comment:
Looks much better with the `<>` convention.

Reply:
But still I am struggling what exactly is now fixing the regression. Can you please elaborate on this?

Reply:
Nvm, I set `profileName` before `imgPath` and `srcPath` got deref.